AppCenter/AppCenterTests/MSDeviceTrackerTests.m

```diff
@@ -246,6 +246,22 @@ - (void)testNoCarrierName {
 }
 #endif
 
+#if TARGET_OS_IOS
+- (void)testNonValidCarrierName {
```

Comment:
format

Reply:
Check with clang_format, no changes from it. Also, all other functions in file are formatted the same way. It looks like it is correct when there is no whitespace between. ")" and function name.